jspdocportal-common-module/src/main/java/org/mycore/jspdocportal/common/bpmn/MCRBPMNMgr.java

```diff
@@ -54,16 +57,43 @@ public class MCRBPMNMgr {
 
     public static final String WF_VAR_DISPLAY_DERIVATELIST = "wfObjectDisplayDerivateList";
 
-    public static final String MCR_BPMN_CONFIG_FILE = "camunda.cfg.xml";
+    public static final String WF_PROCESS_ENGINE_PROPERTIES_PREFIX = "MCR.Worflow.ProcessEngine.";
 
     private static ProcessEngine processEngine;
 
     public static ProcessEngineConfiguration getWorkflowProcessEngineConfiguration() {
         try {
-            return ProcessEngineConfiguration.createProcessEngineConfigurationFromResource(MCR_BPMN_CONFIG_FILE);
+            Map<String, String> props = MCRConfiguration2.getSubPropertiesMap(WF_PROCESS_ENGINE_PROPERTIES_PREFIX);
+            StandaloneProcessEngineConfiguration peConf
+                = (StandaloneProcessEngineConfiguration) ProcessEngineConfiguration
+                    .createStandaloneProcessEngineConfiguration();
+            if (props.containsKey("JdbcDriver")) {
+                peConf.setJdbcDriver(props.get("JdbcDriver"));
+            }
+            if (props.containsKey("JdbcUrl")) {
+                peConf.setJdbcUrl(props.get("JdbcUrl"));
+            }
+            if (props.containsKey("JdbcUsername")) {
+                peConf.setJdbcUsername(props.get("JdbcUsername"));
+            }
+            if (props.containsKey("JdbcPassword")) {
+                peConf.setJdbcPassword(props.get("JdbcPassword"));
+            }
+            if (props.containsKey("DatabaseSchema")) {
+                peConf.setDatabaseSchema(props.get("DatabaseSchema"));
+                // databaseTablePrefix = databaseSchema + '.' 
+                peConf.setDatabaseTablePrefix(props.get("DatabaseSchema") + ".");
+            }
+
+            // default values, currently no support to change via MyCoRe properties
+            peConf.setHistoryTimeToLive("P7D");
+            peConf.setDatabaseSchemaUpdate("true");
```

Comment:
In their [official documentation](https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#schema-generation), the Hibernate team recommends the below:

    Although the automatic schema generation is very useful for testing and prototyping purposes,
    in a production environment, it’s much more flexible to manage the schema using incremental
    migration scripts.

Reply:
I 've used the default value from Camunda, but added possibility to override the values